server/game/cards/01_SOR/units/SpecForceSoldier.ts

```diff
@@ -0,0 +1,26 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { KeywordName, WildcardCardType } from '../../../core/Constants';
+
+export default class SpecForceSoldier extends NonLeaderUnitCard {
```

Comment:
We may need to pause this card for now, sorry I shouldn't have marked as "implementable." The issue is that tracking lose and gain abilities or keywords is kind of complicated and we haven't implemented it yet. For example, the following situation currently doesn't work:

- Echo Base Defender has sentinel
- SpecForce Soldier is played and takes away sentinel
- Protector is attached to Echo Base Defender, giving it sentinel again

Since this is the only card that involves losing a keyword, we probably should not add it until the whole system is working